xlators/mgmt/glusterd/src/glusterd-pmap.h

```diff
@@ -20,38 +20,38 @@
 #include <glusterfs/byte-order.h>
 #include "rpcsvc.h"
 
-struct pmap_port_status {
+struct pmap_ports {
+    int port;
     char *brickname;
     void *xprt;
-    gf_pmap_port_type_t type;
+    struct cds_list_head port_list;
```

Comment:
Any reason for using urcu list definition instead of out own list implementation in `glusterfs/list.h` ?

Reply:
No reason as such. As the urcu lists only were used in the glusterd component, so thought of using the same.